MediaBrowser.Model/Updates/PackageInfo.cs

```diff
@@ -8,55 +9,62 @@ namespace MediaBrowser.Model.Updates
     /// </summary>
     public class PackageInfo
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PackageInfo"/> class.
+        /// </summary>
+        public PackageInfo()
+        {
+            Versions = Array.Empty<VersionInfo>();
+        }
+
         /// <summary>
         /// Gets or sets the name.
         /// </summary>
         /// <value>The name.</value>
-        public string name { get; set; }
+        [JsonPropertyName("name")]
```

Comment:
Instead of adding these attributes everywhere you should be able to tell the JSON serializer to use camelCasing. Like this: https://stackoverflow.com/a/58476761/1988326

Reply:
I know, I just like being explicit

Reply:
Okay sorry, I was thinking that the web API returns in camelCase so we would want to re-use the same configuration, but I didn't realize the API actually uses TitleCase. You're right, better to be explicit here since it deviates from the defaults